app/services/transifex/synchroniser.rb

```diff
@@ -74,7 +74,12 @@ def updated_since_last_pushed?
 
     #Has the resource been updated in Transifex since it was last pulled?
     def translations_updated_since_last_pull?
-      return transifex_service.last_reviewed(@tx_serialisable.tx_slug, @locale) > last_pulled
+      last_reviewed = transifex_service.last_reviewed(@tx_serialisable.tx_slug, @locale)
+      if last_pulled
+        transifex_service.last_reviewed(@tx_serialisable.tx_slug, @locale) > last_pulled
```

Comment:
Should this not just be last_reviewed > last_pulled to avoid doing another GET request?

Reply:
d'oh yes of course - thanks